core/network/address.go

```diff
@@ -765,7 +775,7 @@ func (a addressesPreferringIPv4Slice) Less(i, j int) bool {
 // SortAddresses sorts the given Address slice according to the sortOrder of
 // each address. See Address.sortOrder() for more info.
 func SortAddresses(addrs []SpaceAddress) {
-	sort.Sort(addressesPreferringIPv4Slice(addrs))
+	sort.Sort(addressesPreferringIPv4(addrs))
```

Comment:
Since secondary addresses get sorted with the same order, do we need to make this a `StableSort` instead?

Reply:
I don't think so. `sortOrder` factors the secondary flag (test verifies) and otherwise equal weights are just lexicographically sorted.